drivers/sensor/lsm6dsl/lsm6dsl.c

```diff
@@ -20,11 +21,8 @@
 static inline int lsm6dsl_reboot(struct device *dev)
 {
 	struct lsm6dsl_data *data = dev->driver_data;
-	const struct lsm6dsl_config *config = dev->config->config_info;
 
-	if (i2c_reg_update_byte(data->i2c_master, config->i2c_slave_addr,
-				LSM6DSL_REG_CTRL3_C,
-				LSM6DSL_MASK_CTRL3_C_BOOT,
+	if (data->hw_tf->update_reg(data, LSM6DSL_REG_CTRL3_C, LSM6DSL_MASK_CTRL3_C_BOOT,
```

Comment:
you will need to re-align subsequent lines of parameters. 

On function signature/call, if parameters are broken into different lines, the alignment is made against previous '(' + 1 space.

Apply that everywhere.

Beware of the 80 chars limit btw.

Reply:
I'm pushing again. Now every line is <= than 80 char. And I took care to modify all function calls (at least the one that I changed) with parameter aligned to last '(' + 1 space.